test/support/api/flows.js

```diff
@@ -14,24 +14,27 @@ function flowFixtures() {
     .fixture('test/states').as('fxStates');
 }
 
-function generateData(patients = _.sample(this.fxPatients, 1)) {
+function generateData(patients = _.sample(this.fxPatients, 5)) {
   const data = getResource(_.sample(this.fxFlows, 10), 'flows');
+  const programFlows = _.sample(this.fxProgramFlows, 5);
+  const programActionsSample = _.sample(this.fxProgramActions, 20);
+  const programs = _.sample(this.fxPrograms, 1);
   let included = [];
 
+  included = getIncluded(included, patients, 'patients');
```

Comment:
Moving the included to outside of the each reduces the includes from ~180 to 31.  This is at least a perf improvement